www/js/main.js

```diff
@@ -23,10 +26,54 @@ app.onDeviceReady = function()
 {
 	// The plugin was loaded asynchronously and can here be referenced.
 	ble = evothings.ble;
-
+	navigator.accelerometer.watchAcceleration(onSuccess, onError, {frequency: 75});
   // For development only.
   // toggle_scan(true);
 };
+function resetValues() {
+	throttleValue = 0;
+	steeringValue = 50;
+	update();
+}
+
+function onSuccess(acceleration) {
+    //Mapping steering values acceleration.x between 6 (left) and -6 (right)
+	//Mapping throttle values acceleration.y between 4 (stop) and -4 (full throttle)
+	acceleration.x = -acceleration.x;
+	acceleration.y = -acceleration.y;
+	if(acceleration.x < -6) {
+		acceleration.x = -6;
+	} else if(acceleration.x > 6) {
+		acceleration.x = 6;
+		//Neutral tolerance on direction
+	} else if(acceleration.x < 1 && acceleration.x > -1) {
+		acceleration.x = 0;
+	}
+	if(acceleration.y <= -6) {
+		acceleration.y = -6;
+	} else if(acceleration.y >= 4) {
+		acceleration.y = 4;
+	}
+	//Offset to work on a positive range only.
+	var offset = 6;
+	//Range 12
+	steeringValue = (acceleration.x + offset) * 99 / 12;
+	//Throttle
+	//Range 10
+	//254 is max value
+	throttleValue = (acceleration.y + offset) * 254 / 10;
```

Comment:
Offset permits to work only with positive values, easier for percentages. As min value is -6, we need to shift 6.
Ranges are almost good. 
Maybe we need to modify range on acceleration.x with tolerance. Because between -6 & -1 we should have 49,5%, and same between 1 and 6.